src/mediapipe_internal/mediapipegraphexecutor.cpp

```diff
@@ -938,33 +884,37 @@ Status MediapipeGraphExecutor::partialDeserialize(std::shared_ptr<const ::infere
             return status;
         }
     }
-
     // Validate current timestamp (can be manual or automatic at this point)
     if (!currentStreamTimestamp.IsRangeValue()) {
         SPDLOG_DEBUG("Timestamp not in range: {}", currentStreamTimestamp.DebugString());
         return Status(StatusCode::MEDIAPIPE_INVALID_TIMESTAMP, currentStreamTimestamp.DebugString());
     }
+    return StatusCode::OK;
+}
 
+Status MediapipeGraphExecutor::partialDeserialize(std::shared_ptr<const KFSRequest> request, ::mediapipe::CalculatorGraph& graph) {
```

Comment:
```suggestion
Status MediapipeGraphExecutor::deserializeRequest(std::shared_ptr<const KFSRequest> request, ::mediapipe::CalculatorGraph& graph) {
```

Since we moved from partialSerialize to serializePacket, we might also change the name here